bin/build-contributors.sh

```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env bash
 
+REPLACE_README_MD='s/README.md/docs\/themes\/navy\/layout\/partial\/all-contributors.swig/g'
+REPLACE_CONTRIBUTORS_PER_LINE='s/"contributorsPerLine": 7/"contributorsPerLine": 65535/g'
+REPLACE_CONTRIBUTOR_LINKS='s/<br \/>.*<\/td>/<\/a><\/td>/g'
+
 cp .all-contributorsrc docs/.all-contributorsrc
-sed -i \
-    -e 's/README.md/docs\/themes\/navy\/layout\/partial\/all-contributors.swig/g' \
-    -e 's/"contributorsPerLine": 7/"contributorsPerLine": 65535/g' \
-    docs/.all-contributorsrc
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    sed -i '' -e "$REPLACE_README_MD" -e "$REPLACE_CONTRIBUTORS_PER_LINE" docs/.all-contributorsrc
+else
+    sed -i -e "$REPLACE_README_MD" -e "$REPLACE_CONTRIBUTORS_PER_LINE" docs/.all-contributorsrc
+fi
 
 all-contributors --config docs/.all-contributorsrc generate
-sed -i 's/<br \/>.*<\/td>/<\/a><\/td>/g' docs/themes/navy/layout/partial/all-contributors.swig
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    sed -i '' -e "$REPLACE_CONTRIBUTOR_LINKS" docs/themes/navy/layout/partial/all-contributors.swig
+else
+    sed -i "$REPLACE_CONTRIBUTOR_LINKS" docs/themes/navy/layout/partial/all-contributors.swig
```

Comment:
Same with above comment.

Using variables provides flexibility but I prefer less code and strateforward scripts. they're just scripts right? they're easier to understand with less lines.